### DISCUSSION TITLE
[Projects beta] Task list progress

### DISCUSSION THREAD

**Discussion by Jblew** [community/community#16721] [Projects and Issues] [answered]
Previous projects had a very useful feature of showing how many checkbox tasks are completed. 
![Screenshot 2022-05-16 at 16 19 21](https://user-images.githubusercontent.com/776705/168614069-4426d691-bc0f-4d4f-b905-6ddd1d68e7a0.png)
It would be very useful to have similar display as in previous projects, but would be way more useful to add a Tasks field to the table. This way github beta projects could support epics

*Suggested UI:*
![Screenshot 2022-05-16 at 16 22 06](https://user-images.githubusercontent.com/776705/168614630-93a457cb-7785-4d71-9507-7fac3f05718a.png)

**Comment by funwithtriangles** [comment]
I'd love this for the board view (e.g. the card as pictured above). The table would be nice too.

**Comment by omerbensaadon** [comment, marked as answer]
Hey @Jblew, this is definitely in our future! I would expect this sometime in the next few months. I'll be sure to update this discussion when we release this feature (or something similar) 

EDIT Sep 6: we have this in working software and our testing with Staff users now. I would anticipate getting access to this probably by the end of the year

**Reply by Neudrino** [reply]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

**Reply by JonasWanke** [reply]
Another request for showing the progress on the board view: https://github.com/orgs/community/discussions/28635

**Reply by tac0turtle** [reply]
Was this still planned for end of 2022?

**Reply by oalhait** [reply]
bumping this!

**Reply by Marv93** [reply]
Is there any new information on this?

**Reply by omerbensaadon** [reply]
Hey Tasklists do this now! Docs: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists

We're in Private Beta, you can sign up here: https://github.com/features/issues/signup

**Reply by ipcjs** [reply]
@omerbensaadon 
Are there any requirements for joining the private beta?
I have applied for over a week and still have not received an invitation.

**Reply by yanghua-ola** [reply]
I applied for my org `olachat` a long time ago but haven't got access yet. Would love to use the feature.

**Reply by ndaley7** [reply]
Any Updates?  We are approaching almost a year now since the post.

**Reply by omerbensaadon** [reply]
@ndaley7 what update are you looking for? Is your org not yet flagged in? everyone should be at this point

**Reply by ndaley7** [reply]
I just sent the organization request, but did not know where the feature stood, if everyone is flagged in then I guess I should see it soon.

**Reply by omerbensaadon** [reply]
flagged you in :)

**Reply by mcamurri** [reply]
When is this feature going to exit beta? 

I personally just want back a feature that already existed (the little progress bar when you make a checklist within an issue). 

Also the progress bar below a project description seems to be gone and I found it really useful IMHO.

I also think that asking the user to add extra stuff like the triple backtics and the `[tasklist]` special field when previously you could just create a normal checklist is going to add too much effort for the user and friction to adoption.

Projects are meant to speed up workflow, but if too many features are added, you get the opposite result of slowing it down because of extra overhead from having too many features to choose from.

**Reply by Relequestual** [reply]
@omerbensaadon  I would very much like to enable this beta feature for the `json-schema` org. Thanks =]

**Reply by AndrewDryga** [reply]
@omerbensaadon please roll us in at `firezone` org, would love to test this feature.

**Reply by omerbensaadon** [reply]
Sign up here: https://github.com/features/issues/signup

Will empty the waitlist tomorrow :)

**Reply by AndrewDryga** [reply]
@omerbensaadon Thank you, we got access to it!

I would say that the most desired element was not the tasklist itself (which is great) but also being able to see `X out of Y tasks completed on the project board:
<img width="341" alt="Screenshot 2023-07-12 at 11 23 24" src="https://github.com/community/community/assets/1877644/8c8bb157-6fde-45d3-8ebb-60dbe7cffdd5">

Is it something that we need to enable separately, or it will be rolled out later, maybe I'm doing something wrong so I don't see it?

**Reply by Relequestual** [reply]
@omerbensaadon Hey, hope I haven't missed the boat on this one. Just signed up. Our org has an immediate need of such feature =]

**Reply by yanghua-ola** [reply]
This feature has been immensely helpful even at beta. Just applied for another org for our company. Hope to get access to it soon.

**Reply by morkeleb** [reply]
This is exactly what I need right now, just signed up

**Reply by woody-apple** [reply]
@omerbensaadon I tried to sign up this org: https://github.com/project-chip , but I get the error `Sorry that didn’t work. Member has already been taken.`

Others seemed to work, is there a way to toss our hat in the ring? We're using projects extensively these days. Thanks!

**Reply by yanghua-ola** [reply]
@woody-apple it means someone already signed up for that org. Just need to sit tight and wait.

**Reply by drewvanstone** [reply]
Any updates on when this may exit beta? Our org won't join the private beta, but we're eagerly waiting for this feature.

**Reply by i-balash** [reply]
Any updates??

**Reply by KhaledFastsimon** [reply]
Any Updates

**Reply by i-balash** [reply]
@omerbensaadon any news? The option to show the progress of subtasks in any board would be the missing link in the whole project landscape:

![image](https://github.com/community/community/assets/93594837/e83f1d70-300a-42f8-ba22-64087999e100)

Otherwise Epic-Tasks can not have a clear overview of the progress status. Currently we link Tasks as issues to an Epic-Issue-Task:

![image](https://github.com/community/community/assets/93594837/87f48c0c-2bc0-4413-8674-4a1b4c5b8afc)

but the progress does not appear on other boards:

![image](https://github.com/community/community/assets/93594837/bf92b842-6205-4374-9734-18fae676bf59)

**Reply by dbpolito** [reply]
would love to see this too

**Reply by i-balash** [reply]
@omerbensaadon is there a chance we will see this feature soon?

**Reply by Radai** [reply]
For others looking: I'll guess that this is not coming out anytime soon, checking this issue on the public roadmap: https://github.com/github/roadmap/issues/760 😞 

The history shows:
![image](https://github.com/community/community/assets/709590/b57b712d-64e4-4228-822d-d80139659bb8)

**Reply by gdpdils** [reply]
Any news on this? In 2022, two years ago, it was about to be release "the end of the year". This is one of the most useful tools in a Kanban, it was there the older version, and it's been missing from the new one for years.

**Reply by PPT-DGREEN-GH** [reply]
Any word on this one? We're starting our migration from ADO to GH boards and this seems very important to us.

**Reply by royNiladri** [reply]
From what i understand reading their roadmap for related issues, they had release it at somepoint, got some feedback and moved to subtasks which gives the tracking. Sadly for smaller projects, the ability to quickly create a checklist and track progress would have been great, but I can understand why that might be unwieldy to teams. Still feel, it should have been an option as there are a lot of solo developers out there who would like to track progress in a lightweight fashion.

**Comment by maciej-gurban** [comment]
Being able to group a list of tasks under an "epic" or a "story" is one of the most basic requirements for any project management software. One cannot consider using Github Projects as an alternative to JIRA and similar tools without having this as a feature. Can you please give us an update as to what specifically is the team planning to ship and when?

Having access to what you showcase here seems perfectly enough. Seems that it's been already available to some, but not all. Could you at least make this available again as an opt in?
<img width="427" alt="Screenshot 2024-04-19 at 13 40 54" src="https://github.com/community/community/assets/3663640/18babe39-b930-4640-95fd-52fa7aa79b46">

**Reply by i-balash** [reply]
@omerbensaadon fyi

**Comment by ryu1** [comment]
+1

**Comment by taoliujun** [comment]
Now, no task progress in front of title, it is just an icon.
Another sub-issues progress shown after the title, but create a sub issue is not a good idea for some simple task.

![image](https://github.com/user-attachments/assets/4effc6cc-5b9d-44e5-9835-f9e338e6a171)

![image](https://github.com/user-attachments/assets/6c2ad660-9aff-4533-9c7d-1ff71c10219e)

**Comment by taoliujun** [comment]
no task progress in the issue title now.

**Comment by marinskeh** [comment]
So... it's not possible yet? To to show progress in checked / unchecken tasks in the preview of an issue?
